tests/dualtor_mgmt/test_server_failure.py

```diff
@@ -90,3 +92,42 @@ def lower_tor_mux_state_verfication(state, health):
                       "mux_cable status is unexpected. Should be (standby, unhealthy)")
         pytest_assert(wait_until(30, 1, 0, lower_tor_mux_state_verfication, 'standby', 'unhealthy'),
                       "mux_cable status is unexpected. Should be (standby, unhealthy)")
+
+
+@pytest.mark.enable_active_active
+def test_server_reboot(cable_type, start_icmp_responder, shutdown_icmp_responder,  # noqa: F811
+                       active_active_ports, upper_tor_host, lower_tor_host):       # noqa: F811
+
+    def upper_tor_mux_state_verification(state, health):
+        mux_state_upper_tor = show_muxcable_status(upper_tor_host)
+        return (mux_state_upper_tor[test_iface]['status'] == state and
+                mux_state_upper_tor[test_iface]['health'] == health)
+
+    def lower_tor_mux_state_verfication(state, health):
+        mux_state_lower_tor = show_muxcable_status(lower_tor_host)
+        return (mux_state_lower_tor[test_iface]['status'] == state and
+                mux_state_lower_tor[test_iface]['health'] == health)
```

Comment:
can you try using dualtor test utilities? 

https://github.com/sonic-net/sonic-mgmt/blob/b6d492467abc69c3db32f381cae7ec0bab57ee34/tests/common/dualtor/control_plane_utils.py#L255-L261